lib/go-tc/servers.go

```diff
@@ -286,3 +307,88 @@ type ServerQueueUpdate struct {
 	ServerID util.JSONIntStr `json:"serverId"`
 	Action   string          `json:"action"`
 }
+
+// ServerInterface is the data associated with an interface
+type ServerInterface struct {
+	MaxBandwidth int64  `json:"max_bandwidth" db:"max_bandwidth"`
+	Monitor      bool   `json:"monitor" db:"monitor"`
+	Mtu          int64  `json:"mtu" db:"mtu"`
+	Name         string `json:"name" db:"name"`
+	Server       int    `json:"server" db:"server"`
+}
+
+// ServerIpAddress is the data associated with an IP address
+type ServerIpAddress struct {
+	Address        string `json:"address" db:"address"`
+	Gateway        string `json:"gateway" db:"gateway"`
+	Interface      string `json:"interface,omitempty" db:"interface"`
+	Server         int    `json:"server,omitempty" db:"server"`
+	ServiceAddress bool   `json:"service_address" db:"service_address"`
+}
+
+// ServerInterfaceInfo is the combined data from both IP address information and interface information
+type ServerInterfaceInfo struct {
+	IpAddresses  []ServerIpAddress `json:"ipAddresses" db:"ipAddresses"`
+	MaxBandwidth int64             `json:"max_bandwidth" db:"max_bandwidth"`
+	Monitor      bool              `json:"monitor" db:"monitor"`
+	Mtu          int               `json:"mtu" db:"mtu"`
+	Name         string            `json:"name" db:"name"`
+}
+
+// Value implements the driver.Valuer interface
+// marshals struct to json to pass back as a json.RawMessage
```

Comment:
nit: GoDocs should consist of complete sentences with proper capitalization and terminated by periods.